src/modules/ProductLicense.rb

```diff
@@ -302,12 +307,48 @@ def GetLicenseDialogTerm(languages, license_language, licenses, id)
       )
     end
 
+    # Returns source ID of the base product - initial installation only!
+    # If no sources are found, returns 0.
+    # FIXME: Connected to bsc#993285, refactoring needed
+    #
+    # return [Integer] base_product_id or nil
+    def base_product_id
+      raise "Base product can be only found in installation" unless Stage.initial
+
+      # The first product in the list of known products
+      current_sources = Pkg.SourceGetCurrent(true)
+      current_sources.any? ? current_sources.first : 0
```

Comment:
according doc it should return base_product or nil. So, I guess that ```s/nil/zero/``` in docu.

I'm also curios if ```0``` is somehow special product id ... or why it is used as default.